src/status_im/group_chats/core.cljs

```diff
@@ -21,89 +21,89 @@
   {:events [:chat-removed]}
   [cofx response]
   (fx/merge cofx
-            {:db (dissoc (:db cofx) :current-chat-id)
+            {:db         (dissoc (:db cofx) :current-chat-id)
              :dispatch-n [[:sanitize-messages-and-process-response response]
                           [:pop-to-root-tab :chat-stack]]}
             (notification-center/get-activity-center-notifications-count)))
 
 (fx/defn handle-chat-update
   {:events [:chat-updated]}
-  [_ response]
+  [_ response do-not-navigate?]
   {:dispatch-n [[:sanitize-messages-and-process-response response]
-                [:navigate-chat-updated (.-id (aget (.-chats response) 0))]]})
+                (when-not do-not-navigate? [:navigate-chat-updated (.-id (aget (.-chats response) 0))])]})
 
 (fx/defn remove-member
   "Format group update message and sign membership"
   {:events [:group-chats.ui/remove-member-pressed]}
-  [_ chat-id member]
-  {::json-rpc/call [{:method     "wakuext_removeMemberFromGroupChat"
-                     :params     [nil chat-id member]
+  [_ chat-id member do-not-navigate?]
+  {::json-rpc/call [{:method      "wakuext_removeMemberFromGroupChat"
+                     :params      [nil chat-id member]
                      :js-response true
-                     :on-success #(re-frame/dispatch [:chat-updated %])}]})
+                     :on-success  #(re-frame/dispatch [:chat-updated % do-not-navigate?])}]})
 
 (fx/defn join-chat
   {:events [:group-chats.ui/join-pressed]}
   [_ chat-id]
-  {::json-rpc/call [{:method     "wakuext_confirmJoiningGroup"
-                     :params     [chat-id]
+  {::json-rpc/call [{:method      "wakuext_confirmJoiningGroup"
+                     :params      [chat-id]
                      :js-response true
-                     :on-success #(re-frame/dispatch [:chat-updated %])}]})
+                     :on-success  #(re-frame/dispatch [:chat-updated %])}]})
```

Comment:
`rf/` ??

Reply:
Did not write this code, this is just formatting